test/fixtures.rb

```diff
@@ -255,3 +255,12 @@ class PureValidator
 
   validates :name, presence: true
 end
+
+class UserDefinedValidationTest
+  include Lotus::Validations
+  attr_accessor :foo
+
+  validates :foo, :is_bar do |attribute|
```

Comment:
It would be nice to have a syntax which made using a self defined validator more straight-forward. 

Something like:

```ruby
class BarValidator
  def self.call(attribute)
    attribute.value == 'bar'
   end
end

validates :foo, :is_bar, with: BarValidator 
```